libr/core/cmd_print.c

```diff
@@ -2660,14 +2660,19 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 		return 0;
 	}
 
-	if (mode && mode != ' ') {
+	if (mode && mode != ' '){
 		input++;
 	}
 
 	int w = (input[0] == ' ')
 		? (int)r_num_math (core->num, input + 1)
 		: (int)(core->print->cols * 2.7);
 
+	if (w == 0){
```

Comment:
add a space before {

Reply:
True, I did not realize about those spaces, thanks!